crates/bevy_asset/src/lib.rs

```diff
@@ -46,18 +46,15 @@ pub use loader::*;
 pub use path::*;
 pub use reflect::*;
 
-use bevy_app::prelude::*;
-use bevy_ecs::prelude::*;
+use bevy_app::{prelude::*, MainScheduleOrder};
+use bevy_ecs::schedule::ScheduleLabel;
 
-/// [`SystemSet`]s for asset loading in an [`App`] schedule.
-#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
-#[system_set(base)]
-pub enum AssetSet {
-    /// Asset storages are updated.
-    LoadAssets,
-    /// Asset events are generated.
-    AssetEvents,
-}
+/// Asset storages are updated.
+#[derive(Debug, Hash, PartialEq, Eq, Clone, ScheduleLabel)]
+pub struct LoadAssets;
```

Comment:
I really don't think that either of these need to be schedules. I'm fine to punt to future work if needed though.

Reply:
Yeah I'd prefer to punt this as I think this deserves careful consideration